test/gtest/common/test_helpers.cc

```diff
@@ -685,6 +685,18 @@ const struct sockaddr* sock_addr_storage::get_sock_addr_ptr() const {
     return m_is_valid ? (struct sockaddr *)(&m_storage) : NULL;
 }
 
+const void* sock_addr_storage::get_sock_addr_in_buf() const {
+    const struct sockaddr* saddr = get_sock_addr_ptr();
+
+    EXPECT_NE((const struct sockaddr*)NULL, saddr);
+    EXPECT_TRUE((saddr->sa_family == AF_INET) ||
+                (saddr->sa_family == AF_INET6));
```

Comment:
these should be ucs_assert()

Reply:
is not `ucs_assert` for internal usage only?

Reply:
not only, and the problem here is that EXPECT_EQ does not prevent the execution of following statements